NN_training_example.py

```diff
@@ -26,28 +26,35 @@
           np.arange(-params.matrix_size / 2, params.matrix_size / 2) * params.pixel] for y in
          np.arange(-params.matrix_size / 2, params.matrix_size / 2) * params.pixel])
 
-    #Build NNTrainer
+    # Build NNTrainer
     NN = NNTrainer(params)
 
     # Run NN optimization
+    # Please try running different numbers of epochs (last parameter)
+    # Check the difference in the output for different amounts of training
     trained_model = NN.optimize(LightField(amp, phase), LightField(target, phase), 100)
 
     # Plot loss vs epochs
     NN.plot_loss()
 
-    #Plot the result - optimized phase map
-    plotter = GeneratePropagationPlot(LightField(amp, np.array(trained_model.layers[3].get_weights()[0])), output_type=GeneratePropagationPlot.PLOT_PHASE)
+    # Plot the result - optimized phase map
+    plotter = GeneratePropagationPlot(LightField(amp, np.array(trained_model.layers[3].get_weights()[0])),
```

Comment:
`np.array(trained_model.layers[3].get_weights()[0])` this part is absolutely not readable. I know its phase but why would user know that they should look for phase in there? Return value of the `NN.optimize` needs to be refactored to be more user friendly

Reply:
Good idea. Done.